shader/lighting/tonemapping.frag

```diff
@@ -10,6 +10,31 @@
 
 #include "upscale/lanczos.glsl"
 
+#if defined(COMPUTE)
+layout(local_size_x = 8, local_size_y = 8, local_size_z = 1) in;
+layout(binding = 2) uniform writeonly image2D tonemappedOutput;
+layout(r32f, binding = 3) uniform writeonly image2D hdrLumaOutput;
```

Comment:
`writeonly` doesn't require explicit pixel-format specifier

Reply:
Also compute variant can be removed, if you segregate `sceneTonemapped` (pre CMAA) and anti-aliased `sceneTonemapped`, that is written by `deferred_color_apply_2x2.comp`

Reply:
No, it's the same `sceneTonemapped ` as used in cmaa2

Reply:
In cmaa2_common.glsl;
`layout(binding = 0) uniform sampler2D inputColorReadonly;` - not a storage-image, but a simple texture. Also good to rename it to `sceneTonemapped`, in order to math with c++ side.

In deferred_color_apply_2x2.comp `outputColor` is indeed a writable image, but you dont have to write into a same one.
Maybe deferred_color_apply_2x2 can later be converted into fragment-shader? It's not heavy one compute features, other then a unordered-write.